avocado/plugins/plugins.py

```diff
@@ -58,7 +73,11 @@ def run(self, config):
         for plugins_active, msg in plugin_types:
             LOG_UI.info(msg)
             plugin_matrix = []
-            for plugin in sorted(plugins_active, key=lambda x: x.name):
+            if config.get('plugins.ordered_list'):
+                sorted_plugins = sort_extensions(plugins_active)
+            else:
+                sorted_plugins = sorted(plugins_active, key=lambda x: x.name)
```

Comment:
The same way we would have a `sort_by_priority()` method, I would also suggest to have a `sort_by_name()` method there. So we don't need to sort outside the class scope.